blivet/devices/md.py

```diff
@@ -554,6 +556,42 @@ def preCommitFixup(self):
         if getattr(self.format, "mountpoint", None) == "/boot/efi":
             self.metadataVersion = "1.0"
 
+    def _removeStaleLVM(self):
+        """ Remove any stale LVM metadata that pre-existed in a new array's on-disk footprint.
+
+            XXX This should only be run from _postCreate.
+        """
+        import time
```

Comment:
Why a local import?

Reply:
No good reason. That's just how I added it initially.